Chapters/04-Decomposition-and-ETS.Rmd

```diff
@@ -29,39 +29,39 @@ It is also possible to define mixed models in which, for example, the trend is a
     y_t = (l_{t-1} + b_{t-1}) s_{t-m} \varepsilon_t ,
     (\#eq:MixedAdditiveTrend)
 \end{equation}
```

Comment:
Substitute come with full stop then. Otherwise we need "these", not "These". :)

Reply:
I don't think it needs a full stop after the colon. But point taken about the superfluous comma!